src/appleseed.shaders/src/appleseed/as_metal.osl

```diff
@@ -328,26 +312,6 @@ shader as_metal
         return;
     }
 
-    string distribution;
```

Comment:
That's clearly code that we can remove without breaking anything, regardless of our decision of what to do with the deprecated parameters.